firmware-nucleo/smart-air-hockey-table/App/app_core.c

```diff
@@ -42,12 +43,12 @@ void TIM6_Handler()
 
 void TIM7_Handler()
 {
-//	if (ticks_completed + ticks_missed != ticks_elapsed)
-//	{
-//		uint32_t ticks_difference = ticks_elapsed - ticks_completed - ticks_missed;
-//		ticks_missed += ticks_difference;
-//		// Doesn't do much yet, but we can detect when ticks are missed
-//	}
+	if (ticks_completed + ticks_missed != ticks_elapsed)
+	{
+		uint_fast32_t ticks_difference = ticks_elapsed - ticks_completed - ticks_missed;
+		ticks_missed += ticks_difference;
+		// Doesn't do much yet, but we can detect when ticks are missed
+	}

```

Comment:
where are we using this? not sure why its getting uncommented.

Reply:
Just because we don't use it now doesn't mean it needs to be removed. Most of the drivers are also completely unused at the moment. It'll be useful very soon for integration testing, to make sure our ticks are staying under their time quota.

Reply:
oh I was just curious what / where we are consuming this. Where do we plan to consume this?